src/test/java/org/broadinstitute/hellbender/utils/variant/VariantContextVariantAdapterTest.java

```diff
@@ -39,38 +54,67 @@ public void testVariantAdapter(final List<Variant> expectedVariantList) {
+
+        List<Variant> sameVariants = Lists.newArrayList();
+        Assert.assertTrue(sameVariants.addAll(variantList));
+        DataflowAssert.that(pShards).containsInAnyOrder(sameVariants);
+        p.run();
+    }
+
+
     @DataProvider(name = "VariantDataProvider")
     public Object[][] getVariantData() {
         List<Variant> variantSet = new ArrayList<>();
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 100, 100), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 199, 200), false, true));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 200, 200), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 203, 206), false, true));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 280, 280), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 284, 286), false, true));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 285, 285), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 286, 286), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 999, 999), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 1000, 1000), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 1000, 1003), false, true));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 1076, 1076), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 1150, 1150), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 1176, 1176), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("2", 200, 200), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("2", 525, 525), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("2", 548, 550), false, true));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("2", 640, 640), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("2", 700, 700), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("3", 1, 1), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("3", 300, 300), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("3", 300, 303), false, true));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("3", 400, 400), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("4", 600, 600), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("4", 775, 775), true, false));
-        variantSet.add(new SkeletonVariant(new SimpleInterval("4", 776, 779), false, true));
+        variantSet.add(new SkeletonVariant(new SimpleInterval("1", 100, 100), true, false, defaultUUID()));
```

Comment:
We don't want a counter because we want to compare the sets without UUID (but still use DataflowAssert). So, we need to clear the UUIDs... I don't know a better way.